cmd/snapd-apparmor/export_test.go

```diff
@@ -19,10 +19,21 @@
 
 package main
 
+import "github.com/snapcore/snapd/release"
+
 var (
 	Run                           = run
 	ValidateArgs                  = validateArgs
 	IsContainer                   = isContainer
 	IsContainerWithInternalPolicy = isContainerWithInternalPolicy
 	LoadAppArmorProfiles          = loadAppArmorProfiles
 )
+
+func MockWSL(on_wsl bool) (restorer func()) {
+	old_wsl := release.OnWSL
+
+	release.OnWSL = on_wsl
+	return func() {
+		release.OnWSL = old_wsl
+	}
+}
```

Comment:
The convention is to use camel case. Also, if this is just setting `release.OnWSL` then it doesn't need to be in the `export_test.go`. It can be moved next to the tests.

```suggestion
func MockWSL(onWSL bool) (restorer func()) {
	oldWSL := release.OnWSL

	release.OnWSL = onWSL
	return func() {
		release.OnWSL = oldWSL
	}
}
```